SchedulerBot/SchedulerBot.sln

```diff
@@ -31,6 +29,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SchedulerBot.Business.Entit
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SchedulerBot.Infrastructure.Scheduler", "SchedulerBot.Infrastructure.Scheduler\SchedulerBot.Infrastructure.Scheduler.csproj", "{F27CDE58-A6B6-4BBF-B008-FF05989563DF}"
```

Comment:
I think it might be a good idea to rename the `SchedulerBot.Infrastructure.Scheduler` to `SchedulerBot.Infrastructure.Schedule` since there is no real schedule**r** inside, just things related to the schedule.